src/content/getting-started/cloud-provider-accounts/vmware-cloud-director/_index.md

```diff
@@ -0,0 +1,62 @@
+weight: 80
+user_questions:
+  - What are the VMware Cloud Director prerequisites for use with Giant Swarm?
+  - What do I need to configure in VMware Cloud Director in order to run Giant Swarm clusters?
+  - What user permissions do I need to run Giant Swarm clusters in VMware Cloud Director?
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-clouddirector
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-10-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+In order to run the Giant Swarm platform in your VMware Cloud Director (VCD) environment, a number of prerequisites must be satisfied to support Cluster API Provider VMware Cloud Director (CAPVCD).
+
+## Cloud Director infrastructure
+
+VMware Cloud Director must run with API version 36 compatibility at least. However, it is highly recommended to run VCD 10.4 or above as it includes shared virtual service IPs.
+
+## VCD User permissions
+
+The CAPVCD controller doesn’t require to authenticate directly with the VCD API. This is done on a “per cluster” level as the user will specify a secret containing their credentials in the cluster manifest/app. This is so the VCD Resource quotas of each user apply when deploying clusters.
+
+The user configured for CAPVCD must have at least the following permissions attached to its user role in VCD:
+
+* Rights from the default vApp Author role
+* User > Manage user's own API token
+* vApp > Preserve All ExtraConfig Elements During OVF Import and Export
+* Gateway > View Gateway
+* Gateway Services > NAT Configure, LoadBalancer Configure
+* Organization VDC => Create a Shared Disk
+* User > Manage user's own API token
+
+## Networking
+
+The infrastructure backing the OVDC where the clusters will be deployed must be backed by VMware NSX-T and NSX Advanced Load Balancer (ALB). The deprecated NSX-V is not supported.
+
+The Load Balancer section must be enabled on the Edge gateway with a Service Engine Group (SEG). Note that it is recommended to set a dedicated SEG to properly isolate tenants from each other. A pool of external IPs must be available on the Edge gateway to create load balancers.
+
+The nodes need at least one network with DHCP or a static IP pool enabled to obtain an IP when spinning up. They must have access to the VCD API endpoint for the controllers to work (CAPVCD, cloud provider interface and container storage interface). Note that we support connecting multiple networks to the nodes with static routes.
+
+Access to the internet is required for things like Flux pulling artifacts from Github, reaching out to Lets Encrypt to validate DNS01 challenges or containerd pulling container images. We support HTTP proxies and we can provide the list of domains to whitelist on the corporate firewall. Also, Giant Swarm is working on a local artifact registry as part of our efforts to reduce the need for internet connectivity.
```

Comment:
I removed the roadmap item since that shouldn't go into a getting-started article IMO, creating future assumptions that may or may not hold true. Regarding the allowlist, is it public so we can share the link? Will you add documentation to https://docs.giantswarm.io/advanced/private-clusters/ to describe the proxy feature?

```suggestion
Access to the internet is required for things like Flux (our continuous delivery tooling) pulling artifacts from GitHub, reaching out to Let's Encrypt for certificate issuing, or containerd pulling container images. We support HTTP proxies and can provide the list of domains to allowlist on the corporate firewall.
```